.github/workflows/e2e.yml

```diff
@@ -31,8 +31,7 @@ jobs:
       - uses: haya14busa/action-workflow_run-status@967ed83efa565c257675ed70cfe5231f062ddd94
       - uses: actions/checkout@v2.3.4
         with:
-          repository: data-integrations/servicenow-plugins
-          path: plugin
```

Comment:
Why do we need to remove these lines?

Reply:
We are using ref so we don't really need these lines.

Reply:
We still need to specify the `path` since it is being used by the `run_e2e_test` script

Reply:
Added.